classes/publish/package/tasks/upload-files.js

```diff
@@ -52,7 +53,7 @@ module.exports = class UploadFiles extends Task {
                         'Server was unable to write file to storage',
                     );
                 default:
-                    throw new Error('Server failed');
+                    throw new Error(err.message);
```

Comment:
What do you think about decorating the server error with addition info? Something like:

```suggestion
                    throw new Error(`Upload to server failed, server responded with: ${err.message}`);
```